instrumentation/src/main/java/io/opentelemetry/android/OpenTelemetryRumBuilder.java

```diff
@@ -413,6 +436,7 @@ private SdkTracerProvider buildTracerProvider(SessionId sessionId, Application a
                         .addSpanProcessor(new SessionIdSpanAppender(sessionId));
 
         SpanExporter spanExporter = buildSpanExporter();
+        initializationEvents.spanExporterInitialized(spanExporter);
```

Comment:
I was having issues to verify that the right type of exporter (the disk exporter) was properly set in the unit tests, so one of the least ugly ways I could manage to do so was by adding this new function to the InitializationEvents interface and also a package-private setter for it below in this class (to set a mock impl from the tests). I'm open to better alternatives I might have missed 🙂

Reply:
Maybe `initializationEvents` takes `spanExporter` in the ctor? (L331) so if you can provide/mock the `spanExporter`, you don't need a setter.

Reply:
What if, instead of creating an init event for the span exporter, there was an init event for the disk buffering being initialized? That's really the feature here, and then you can assert that it was called I guess?

Reply:
> What if, instead of creating an init event for the span exporter, there was an init event for the disk buffering being initialized? That's really the feature here, and then you can assert that it was called I guess?

I see what you mean, although the initialization of the disk buffering tool doesn't say anything about the feature actually being used in the OpenTelemetry instance that has been built here, which I believe should be what we verify to guarantee that the RUM config is honored, and all those pieces are out of the scope of the disk buffering feature.

Reply:
ok, I see the issue, since GitHub collapsed the code diff, I thought that these changes and the changes on L331 were a single method, so you could have used a ctor, my bad.